LoopKitUI/Views/InstructionList.swift

```diff
@@ -42,23 +48,37 @@ public struct InstructionList: View {
                         .foregroundColor(Color.white)
                         .font(.caption)
                         .accessibility(label: Text("\(index+1), ")) // Adds a pause after the number
-                    instructionView(instructions[index])
+                    instructionView(instructions[index], markdownEnabled: markdownEnabled)
                 }
                 .accessibilityElement(children: .combine)
             }
         }
     }
     
     @ViewBuilder
-    private func instructionView(_ instruction: Instruction) -> some View {
+    private func instructionView(_ instruction: Instruction, markdownEnabled: Bool = false) -> some View {
         VStack(alignment: .leading, spacing: 2) {
-            Text(instruction.text)
-                .fixedSize(horizontal: false, vertical: true)
-            if let subtext = instruction.subtext {
-                Text(subtext)
+            
+            if markdownEnabled {
+                Text(LocalizedStringKey(instruction.text))
```

Comment:
LocalizedStringKey looks up strings for localization in the current module's bundle; in this case, LoopKit.  The localization lookup for string that originate in a different module should be handled in that module.

Reply:
Was this informational only, or did you want me to do any change there?

Reply:
This will not work for localization.

Reply:
One option might be to change this class to accept `LocalizedStringKey` for instruction text & subtext, instead of `String`.  The localization needs to happen in the caller's framework, though.